pycbc/workflow/matched_filter.py

```diff
@@ -293,25 +294,27 @@ def setup_matchedfltr_dax_generated_multi(workflow, science_segs, datafind_outs,
 
     logging.info("Setting up matched-filtering for %s." %(' '.join(ifos),))
 
-    if match_fltr_exe == 'lalapps_coh_PTF_inspiral':
-        from pylal.legacy_ihope import select_legacy_matchedfilter_class
-        from pycbc.workflow.grb_utils import get_sky_grid_scale
-        exe_class = select_legacy_matchedfilter_class(match_fltr_exe)
-        cp.set('inspiral', 'right-ascension', cp.get('workflow', 'ra'))
-        cp.set('inspiral', 'declination', cp.get('workflow', 'dec'))
-        if cp.has_option("jitter_skyloc", "apply-fermi-error"):
-            cp.set('inspiral', 'sky-error',
-                   str(get_sky_grid_scale(float(cp.get('workflow',
-                                                       'sky-error')))))
-        else:
-            cp.set('inspiral', 'sky-error',
-                   str(get_sky_grid_scale(float(cp.get('workflow',
-                                                       'sky-error')),
-                                          sigma_sys=0.0)))
-        cp.set('inspiral', 'trigger-time', cp.get('workflow', 'trigger-time'))
-        cp.set('inspiral', 'block-duration',
-               str(abs(science_segs[ifos[0]][0]) - \
-                       2 * int(cp.get('inspiral', 'pad-data'))))
+    if match_fltr_exe == 'pycbc_multi_inspiral':
+        exe_class = select_matchedfilter_class(match_fltr_exe)
+        cp.set('inspiral', 'longitude',\
+               str(radians(float(cp.get('workflow', 'ra')))))
+        cp.set('inspiral', 'latitude',\
+               str(radians(float(cp.get('workflow', 'dec')))))
+        # from pycbc.workflow.grb_utils import get_sky_grid_scale
```

Comment:
Remove unneeded old code.

Reply:
I wasn't sure whether this might be needed when we add in searching over a sky grid? Should I just delete it?

Reply:
If this is being left in for a reason, then please indicate that as an in-line comment.